sources/us/wi/manitowoc.json

```diff
@@ -16,10 +16,12 @@
         "number": "HNUM",
         "street": [
             "PREDIR",
+            "PRE_CNTY",
```

Comment:
I think this is the county's road name and might not be useful for us here. Or at least it looks like a different road name in some cases.

Reply:
The PRE_CNTY field contains values like "CTH", "STH", and "USHY" functioning as a street prefix. The NAME field when PRE_CNTY is not empty contains the road number or letter. For [state highway 42](http://webmap2.manitowoc-county.com/agsweb/rest/services/Web_MercParBaseSandBox/MapServer/84/query?where=1%3D1+and+pre_cnty+%3D+%27STH%27+and+name+%3D+%2742%27&text=&objectIds=&time=&geometry=&geometryType=esriGeometryEnvelope&inSR=&spatialRel=esriSpatialRelIntersects&relationParam=&outFields=*&returnGeometry=true&returnTrueCurves=false&maxAllowableOffset=&geometryPrecision=&outSR=&returnIdsOnly=false&returnCountOnly=false&orderByFields=&groupByFieldsForStatistics=&outStatistics=&returnZ=false&returnM=false&gdbVersion=&returnDistinctValues=false&resultOffset=&resultRecordCount=100&f=html), PRE_CNTY contains "STH" and NAME contains "42". Similarly, for [county road Q](http://webmap2.manitowoc-county.com/agsweb/rest/services/Web_MercParBaseSandBox/MapServer/84/query?where=1%3D1+and+pre_cnty+%3D+%27CTH%27+and+name+%3D+%27Q%27&text=&objectIds=&time=&geometry=&geometryType=esriGeometryEnvelope&inSR=&spatialRel=esriSpatialRelIntersects&relationParam=&outFields=*&returnGeometry=true&returnTrueCurves=false&maxAllowableOffset=&geometryPrecision=&outSR=&returnIdsOnly=false&returnCountOnly=false&orderByFields=&groupByFieldsForStatistics=&outStatistics=&returnZ=false&returnM=false&gdbVersion=&returnDistinctValues=false&resultOffset=&resultRecordCount=100&f=html), PRE_CNTY contains "CTH" and NAME contains "Q".

Are there any particular records that stood out to you as incorrect? I'm happy to profile further.

Reply:
Ah, thanks for the explanation! I was looking at [the sample](https://s3.amazonaws.com/data.openaddresses.io/runs/564060/sample.json) but misread it.